test-example-diagrams/src/main/resources/timer-event/TimerEvent.md

```diff
@@ -0,0 +1,18 @@
+# Receive Event
```

Comment:
Here, the header should be **Timer Event**

Reply:
Done